### DISCUSSION TITLE
error: select "collection", "singleton", "note", "sort_field", "accountability" from "directus_collections" - relation "directus_collections" does not exist

### DISCUSSION THREAD

**Discussion by sambitraze** [directus/directus#16245] [Archive]
i am running npx directus bootstrap command but I am getting this issue

error: select "collection", "singleton", "note", "sort_field", "accountability" from "directus_collections" - relation "directus_collections" does not exist

**Comment by rijkvanzanten** [comment]
Sounds like you don't have the system tables installed. Try running `npx directus init`

**Reply by sambitraze** [reply]
i already have a snapshot.yaml

**Reply by sambitraze** [reply]
even thought I run npx directus init it giving me the same error

**Reply by rijkvanzanten** [reply]
No idea. All I know based on the limited info available is that that error most likely means it's trying to start up the API without the system tables existing in the database you're connecting to

**Reply by sambitraze** [reply]
ya, that I also know but u see I have used the same configuration and methods for other projects it's working .. I will keep trying to find out solution meanwhile waiting for others to comment on this

**Reply by rijkvanzanten** [reply]
> I have used the same configuration and methods for other projects it's working

That's a paradox; if you've done the exact same steps, you'd get the same results 🙂

**Reply by sambitraze** [reply]
mismatched env versions. My bad!😢

**Reply by jclaveau** [reply]
I have the same error and can't figure why. Do you remember what you had to change to make it work?

**Reply by jclaveau** [reply]
In case somebody has the same issue than me, it occurred in my case due to an extension hook running `const collections: Collection[] = await collectionsService.readByQuery();` during its initialization. Cheers
